siteorigin-panels.php

```diff
@@ -327,7 +328,7 @@ public function generate_post_content( $content ) {
 						$content = explode( $matches[0], $content, 2 );
 						$content = $content[0];
 						$content = force_balance_tags( $content );
-						if ( ! empty( $matches[1] ) && ! empty( $more_link_text ) ) {
+						if ( ! empty( $matches[1] ) ) {
```

Comment:
Took a little while to follow what was happening here, so some comments about how it's trying to figure out what to display as the "Read more" text would be nice, but obviously not a blocker.